src/pages/BorrowerProfile/BorrowerProfile.vue

```diff
@@ -487,6 +489,7 @@ export default {
 						client.query({ query: experimentAssignmentQuery, variables: { id: SHARE_LANGUAGE_EXP } }),
 						client.query({ query: experimentAssignmentQuery, variables: { id: FIVE_DOLLARS_NOTES_EXP } }),
 						client.query({ query: experimentAssignmentQuery, variables: { id: EDUCATION_PLACEMENT_EXP } }),
+						client.query({ query: experimentAssignmentQuery, variables: { id: ACTIVITY_FEED_EXP } }),
```

Comment:
I think is needed no more...
@dyersituations could have more context.

Reply:
Correct, this line isn't needed, since we assign all active experiments globally based on the Kiva Admin setting.

Reply:
Maybe a tech debt removing all those cases...

Reply:
Yeah we can leave the extra ones for now, so we don't have to functional verify everything.